go.mod

```diff
@@ -18,5 +18,6 @@ require (
 	github.com/stretchr/testify v1.6.1 // indirect
 	golang.org/x/sys v0.0.0-20201013132646-2da7054afaeb
 	gopkg.in/cheggaaa/pb.v1 v1.0.28
+	gopkg.in/yaml.v2 v2.3.0
```

Comment:
I see we add back in yaml.v2 as a dependency next to yaml.v3. Is there a reason we can't use yaml.v3 in these tasks?

Reply:
sneaky little dependency! I had removed it from `go.mod` and ran `go mod tidy` file but then, of course, Go modules brought it back because I had forgotten to remove it from the import section on top of the file task😆